vendor/github.com/containers/common/pkg/seccomp/types.go

```diff
@@ -6,8 +6,12 @@ package seccomp
 
 // Seccomp represents the config for a seccomp profile for syscall restriction.
 type Seccomp struct {
-	DefaultAction   Action `json:"defaultAction"`
+	DefaultAction Action `json:"defaultAction"`
+
+	// DefaultErrnoRet is obsolete, please use DefaultErrno
```

Comment:
Do we need to update our seccomp handling in `chroot` to account for this?

Reply:
@giuseppe PTAL